### ISSUE TITLE
Jekyll 3.0.0.beta1 - jekyll/deprecator (LoadError)

### ISSUE THREAD

**Issue by kaatt** [jekyll/jekyll#3353]
My Jekyll site uses `jekyll-redirect-from` gem and a couple others. Simply installing `v3.0.0.beta1` and running `jekyll b` didn't work:
> /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/specification.rb:2007:in `raise_if_conflicts': Unable to activate jekyll-redirect-from-0.6.2, because jekyll-3.0.0.pre.beta1 conflicts with jekyll (~> 2.0) (Gem::LoadError)

Thanks to @envygeeks in #3324, I forked the gems, edited the `.gemspec` file and commented out the  `s.add_dependency "jekyll"` line.

Now running any `jekyll` command gives me this:
> fatal: Not a git repository (or any of the parent directories): .git
/Library/Ruby/Gems/2.0.0/gems/jekyll-3.0.0.pre.beta1/bin/jekyll:15:in `<top (required)>': cannot load such file -- jekyll/deprecator (LoadError)
	from /usr/bin/jekyll:23:in `load'
	from /usr/bin/jekyll:23:in `<main>'

Everything works if I don't put the lines in `_config.yml` to use extra gems.

**Comment by parkr** [comment]
If you have a `Gemfile`, run `bundle exec jekyll  build`. Using `bundle exec` is highly recommended.

**Comment by kaatt** [comment]
Thanks, I fixed this by adding `gem 'jekyll', '3.0.0.pre.beta1'` to my `Gemfile`.

**Comment by kaatt** [comment]
Everything works but I get this:
```
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/json/version.rb:3: warning: already initialized constant JSON::VERSION
/Library/Ruby/Gems/2.0.0/gems/json-1.8.2/lib/json/version.rb:3: warning: previous definition of VERSION was here
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/json/version.rb:4: warning: already initialized constant JSON::VERSION_ARRAY
/Library/Ruby/Gems/2.0.0/gems/json-1.8.2/lib/json/version.rb:4: warning: previous definition of VERSION_ARRAY was here
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/json/version.rb:5: warning: already initialized constant JSON::VERSION_MAJOR
/Library/Ruby/Gems/2.0.0/gems/json-1.8.2/lib/json/version.rb:5: warning: previous definition of VERSION_MAJOR was here
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/json/version.rb:6: warning: already initialized constant JSON::VERSION_MINOR
/Library/Ruby/Gems/2.0.0/gems/json-1.8.2/lib/json/version.rb:6: warning: previous definition of VERSION_MINOR was here
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/json/version.rb:7: warning: already initialized constant JSON::VERSION_BUILD
/Library/Ruby/Gems/2.0.0/gems/json-1.8.2/lib/json/version.rb:7: warning: previous definition of VERSION_BUILD was here
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/json/common.rb:99: warning: already initialized constant JSON::NaN
/Library/Ruby/Gems/2.0.0/gems/json-1.8.2/lib/json/common.rb:99: warning: previous definition of NaN was here
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/json/common.rb:101: warning: already initialized constant JSON::Infinity
/Library/Ruby/Gems/2.0.0/gems/json-1.8.2/lib/json/common.rb:101: warning: previous definition of Infinity was here
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/json/common.rb:103: warning: already initialized constant JSON::MinusInfinity
/Library/Ruby/Gems/2.0.0/gems/json-1.8.2/lib/json/common.rb:103: warning: previous definition of MinusInfinity was here
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/json/common.rb:128: warning: already initialized constant JSON::UnparserError
/Library/Ruby/Gems/2.0.0/gems/json-1.8.2/lib/json/common.rb:128: warning: previous definition of UnparserError was here
```
